### DISCUSSION TITLE
Support for JSR?

### DISCUSSION THREAD

**Discussion by MariusVatasoiu** [esm-dev/esm.sh#797] [Ideas] [answered]
Since JSR introduced a new scheme for imports (jsr:) and all dependencies of a package are loaded using the same scheme, by default, packages are no longer compatible with browsers. 

It feels a good case esm.sh to transpile packages from jsr: imports to URLs.

**Comment by ije** [comment, marked as answer]
i'm look into it!

**Reply by MariusVatasoiu** [reply]
great! 🎉

**Reply by kaleidawave** [reply]
It works now with `https://esm.sh/jsr/@scope/package@0.0.0`

**Comment by renhiyama** [comment]
@ije I have already discussed with jsr devs and created a working function that can import jsr packages via URL imports. If you want to, lmk and I will send a link of the code.
